src/main/java/io/fusionauth/domain/event/EventType.java

```diff
@@ -50,6 +50,10 @@
 
   UserCreateComplete("user.create.complete"),
 
+  UserIdentityProviderLink("user.idp.link"),
+
+  UserIdentityProviderUnlink("user.idp.unlink"),
+
```

Comment:
1. Please alphabetize these
2. I wonder if we should spell out `idp` - other events such as `user.two-factor.method.add` use the long form and mostly matches the Enum name. Would it be too long to do `user.identity-provider.link` and `user.identity-provider.unlink`?